packages/forms-web-app/__tests__/unit/controllers/examination/session/examination-session.test.js

```diff
@@ -154,6 +156,28 @@ describe('controllers/examination/session/examination-session', () => {
 			});
 		});
 	});
+	describe('#getExaminationSubmissionItems', () => {
+		describe('When getting submission items', () => {
+			const mockSession = {
+				examination: { submissionItems: ['mock submission item 1', 'mock submission item 2'] }
+			};
+			const result = getExaminationSubmissionItems(mockSession);
+			it('should return the submission items', () => {
+				expect(result).toEqual(['mock submission item 1', 'mock submission item 2']);
+			});
+		});
+	});
+	describe('#getExaminationSubmissionItemsLength', () => {
```

Comment:
whats the point in this test ? 

why not just use length inline with base session function ?